README.md

```diff
@@ -86,164 +86,166 @@ Some additionally important notes about how snap works:
-        /intel/mock:
-          name: "root"
-          password: "secret"
-      process:
-        -
-          plugin_name: "passthru"
-          config:
-            debug: true
-          process: null
-          publish:
-            -
-              plugin_name: "mock-file"
-              config:
-                file: "/tmp/snap_published_mock_file.log"
-                debug: true
-```
 
-and then start the task:
+Tasks can be created directly via `snapctl`, but due to length of task it makes more sense to load from a `Task Manifest` written in JSON or YAML format.
 
+Now, we will download and load the [psutil Task Manifest](examples/psutil-file.yaml):
 ```
-$ cd ~/snap
-$ snapctl task create -t mock-file.yaml
+$ curl https://raw.githubusercontent.com/intelsdi-x/snap/master/examples/tasks/psutil-file.yaml -o /tmp/psutil-file.yaml
+$ snapctl task create -t /tmp/psutil-file.yaml
 Using task manifest to create task
 Task created
 ID: 8b9babad-b3bc-4a16-9e06-1f35664a7679
 Name: Task-8b9babad-b3bc-4a16-9e06-1f35664a7679
 State: Running
 ```
 
-This task generates mock data, "processes" it through a passthrough mechanism and then publishes it to a file. Now with a running task, you should be able to do two things:
+NOTE: in subsequent commands use the task ID from your CLI output, not the example task ID shown below.
 
-See the data that is being published to the file:
+This starts a task collecting metrics via psutil, then publishes the data to a file. To see the data published to the file (CTRL+C to exit):
 ```
 $ tail -f /tmp/snap_published_mock_file.log
-^C
 ```
 
-Or tap into the data that Snap is collecting using the Task ID to watch the task (note: your Task ID will be different):
+Or directly tap into the data stream that Snap is collecting using `snapctl task watch` (
 ```
 $ snapctl task watch 8b9babad-b3bc-4a16-9e06-1f35664a7679
+NAMESPACE                             DATA             TIMESTAMP
+/intel/psutil/cpu/cpu-total/idle      451176.5         2016-10-14 11:01:44.666137773 -0700 PDT
+/intel/psutil/cpu/cpu-total/system    33749.2734375    2016-10-14 11:01:44.666139698 -0700 PDT
+/intel/psutil/cpu/cpu-total/user      65653.2578125    2016-10-14 11:01:44.666145594 -0700 PDT
+/intel/psutil/load/load1              1.81             2016-10-14 11:01:44.666072208 -0700 PDT
+/intel/psutil/load/load15             2.62             2016-10-14 11:01:44.666074302 -0700 PDT
+/intel/psutil/load/load5              2.38             2016-10-14 11:01:44.666074098 -0700 PDT
 ```
 
-If the Task ID already scrolled by, you can list it with:
-```
-$ snapctl task list
-```
-
-Nice work - you're all done with this example. You ran `snapd` manually, so stopping the daemon process will stop any running tasks and unload any plugins we loaded.
+Nice work - you're all done with this example. You started `snap-telemetry` service earlier, so use the appropriate service command to stop the daemon, or run the following command if you ran `snapd` manually:
```

Comment:
I feel like the condition is flipped in this sentence.  Maybe something like?

```
If you started `snap-telemetry` as a service earlier, use the appropriate service command to stop the daemon. If you ran `snapd` manually use:
```

Or alternatively something like:
```
If you started `snap-telementry` service earlier, use the appropriate service command to stop the daemon, or run the following command if you ran `snapd` manually:
```